applications/MeshingApplication/custom_processes/mmg_process.h

```diff
@@ -349,6 +308,8 @@ class KRATOS_API(MESHING_APPLICATION) MmgProcess
     Parameters mThisParameters;                                      /// The parameters (can be used for general pourposes)
     NodeType::DofsContainerType  mDofs;                              /// Storage for the dof of the node
 
+    MmgUtilities<TMMGLibray> mMmmgUtilities;                         /// The MMG utilities class
```

Comment:
`MmgUtilities` has no state. Why not just create it in local scope when you need it?

Reply:
State you mean member variables?, yes it does, but in the cpp due to linking problems with the MMG library

Reply:
Yes, that's what I meant. My point was, why not just construct it on the fly where you need it since there is nothing stored in that object?

Reply:
The member variables are on the cpp, it is because a linking problem

Reply:
I see that you have it in the cpp, but it is not a member variable. Thus making mMmmgUtilities a member is odd as it doesn't serve any purpose, but aside from the confusion it is harmless so if you want to keep it this way, I won't argue about it. It's just weird ...